### PR TITLE
Add page for JS benchmarks

### PR DESCRIPTION
**Pull request by martinkuba**
The Javascript SDK now has microbenchmark tests that are automatically run on every push to main. The results are currently published at https://open-telemetry.github.io/opentelemetry-js/benchmarks/.

This PR adds a page here with the same results. This is similar to what has already been done for the Collector load tests.
https://opentelemetry.io/docs/collector/benchmarks/

### CODE REVIEW

**Comment by cartersocha** [comment]
@chalin could you take a look at the errors please?

**Comment by cartermp** [comment]
In this case you can look at the logs to see what failed, an `npm run fix` should take care of it.

**Comment by martinkuba** [comment]
@cartermp It looks like the spelling is failing on words that are part of the JS code. Is there a way to bypass that?

**Comment by cartermp** [comment]
Hrm. That should have fixed the spelling thing. Ugh, sorry.

In the yaml metadata at the top of the file, you can add:

```
cSpell:ignore: rrggbbaa Elems
```

File: content/en/docs/instrumentation/js/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+---
+title: Benchmarks
+weight: 99
```
Comment (svrnm) [inline content/en/docs/instrumentation/js/benchmarks.md:3]: Can you do a weight above 100? (see https://github.com/open-telemetry/opentelemetry.io/issues/2829), serverless has 100 right now, so either swap it or pick 101 or 110

**Review by svrnm — APPROVED** [review]
lgtm, only a minor change request.

Since this is the second benchmark page we should consider moving the CSS to a central place for that (maybe that also works for the JS piece?), not necessarily a part of this PR, but let's have a follow up issue

**Review by svrnm — APPROVED** [review]

**Pull request merged**